vtm-android/src/org/oscim/android/MapView.java

```diff
@@ -180,6 +184,25 @@ public int getHeight() {
             return mMapView.getHeight();
         }
 
+        @Override
+        public int getScreenWidth() {
+            return getScreenSize().x;
+        }
+
+        @Override
+        public int getScreenHeight() {
+            return getScreenSize().y;
+        }
+
+        private Point getScreenSize() {
+            WindowManager wm = (WindowManager) mMapView.getContext()
+                    .getSystemService(Context.WINDOW_SERVICE);
+            Display display = wm.getDefaultDisplay();
+            Point size = new Point();
```

Comment:
Also `Point` creation can be moved in class level to not be created with each call.